internal/resources/providers/awslib/iam/user.go

```diff
@@ -46,7 +46,6 @@ const (
 func (p Provider) GetUsers(ctx context.Context) ([]awslib.AwsResource, error) {
 	apiUsers, err := p.listUsers(ctx)
 	if err != nil {
-		p.log.Errorf("fail to list users, error: %v", err)
 		return nil, err
```

Comment:
Why not wrap this one too?

Reply:
It's wrapped by `listUsers()`, too many wraps 🌯🫃